src/routes/Player/ControlBar/SeekBar/SeekBar.js

```diff
@@ -56,7 +58,13 @@ const SeekBar = ({ className, time, duration, buffered, onSeekRequested }) => {
                 onSlide={onSlide}
                 onComplete={onComplete}
             />
-            <div className={styles['label']}>{formatTime(duration)}</div>
+            <Button onClick={toggleRemainingTimeMode}>
+                <div className={styles['label']}>
+                    {remainingTimeMode
```

Comment:
```suggestion
                    {remainingTimeMode && duration !== null && !isNaN(duration)
```

Reply:
Add this check too to prevent this --> 

https://github.com/Stremio/stremio-web/assets/117831817/c0fa0d09-ffe8-4c47-bfd4-468fe96b2e05

Reply:
@kKaskak accepted your commit suggestion.